apps/events/serializers/events.py

```diff
@@ -5,32 +5,12 @@
 from rest_framework import serializers
 
 from apps.events.models import Event, Tag, Category, Schedule, Currency
+from apps.events.serializers import city as city_serializers
 from apps.profiles.models import User
 from apps.chats.models import Chat
 from . import currency, utils
-
-
-class LocationSerializer(serializers.ModelSerializer):
-    latitude = serializers.DecimalField(
-        max_value=180, min_value=-180,
-        write_only=True, max_digits=18,
-        decimal_places=15,
-    )
-    longitude = serializers.DecimalField(
-        max_value=180, min_value=-180,
-        write_only=True, max_digits=18,
-        decimal_places=15,
-    )
-
-    class Meta:
-        model = Event
-        fields = ["latitude", "longitude"]
-
-    def to_representation(self, value):
-        return {
-            "latitude": value.y,
-            "longitude": value.x
-        }
+from .city import LocationSerializer
+from ..models.city import City
```

Comment:
if you mix imports it might look better to do:
from ..models.city import City -> from apps.events.models.city import City
and 
from .city import LocationSerializer -> from . import city